src/Documentation/Getting-Started-With-Orleans/toc.yml

```diff
@@ -4,10 +4,6 @@
   href: Developing-a-Grain.md
 - name: Developing a Client
   href: Developing-a-Client.md
-- name: Grain Persistence
-  href: Grain-Persistence.md
-- name: Observers
-  href: Observers.md
 - name: Running the Application
   href: Running-the-Application.md
 - name: Silos
```

Comment:
I think the order here can be 
 - Grains
 - Grain lifecycle
 - Developping a grain
- Clients
- Developping a client
- Silos
- Running application
- Debugging

I suggested this to group strongly coupled documents as close to each other as possible. For example, Grain, Developing a grain, Grain lifecycle should be listed next to each other since they are strongly coupled. So user can go from Grain, to its development and its lifecycle. The same go with Client. 

And then after we finished explain concept about grain, silos and client, then we can go to topics about running the application and debugging.